bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/thing/channel-group-types.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tplinkrouter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="wifiGroupType">
+		<label>Wifi</label>
+		<description>Wifi channels</description>
+		<channels>
+			<channel id="status" typeId="switch-type">
+				<label>Wifi Status</label>
+				<description>Wifi status</description>
+			</channel>
+			<channel id="ssid" typeId="string-type">
+				<label>Wifi SSID</label>
+				<description>Wifi SSID</description>
+			</channel>
+			<channel id="bandwidth" typeId="bandwidth-type">
+				<label>Wifi BandWidth</label>
+				<description>Wifi bandwidth</description>
+			</channel>
+			<channel id="qss" typeId="switch-type">
+				<label>Wifi QSS</label>
+				<description>Wifi QSS</description>
+			</channel>
+			<channel id="secMode" typeId="security-mode-type">
+				<label>Wifi Security Mode</label>
+				<description>Wifi Security Mode</description>
+			</channel>
+			<channel id="authentication" typeId="authentication-type">
+				<label>Wifi Authentication Mode</label>
+				<description>Wifi Authentication Mode</description>
+			</channel>
+			<channel id="encryption" typeId="encryption-type">
+				<label>Wifi Encryption Mode</label>
+				<description>Wifi Encryption Mode (only for PSK security mode)</description>
+			</channel>
+			<channel id="key" typeId="string-type">
+				<label>Wifi Key</label>
+				<description>Wifi Key</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="switch-type">
+		<item-type>Switch</item-type>
+		<label>TpLink Switch Type</label>
+		<description>Switch Type for TpLinkrRouter Binding</description>
+	</channel-type>
+	<channel-type id="string-type">
+		<item-type>String</item-type>
+		<label>TpLink String Type</label>
+		<state pattern="%s" readOnly="true"/>
+	</channel-type>
+	<channel-type id="security-mode-type">
+		<item-type>String</item-type>
+		<label>Wifi Security Mode Type</label>
+		<state readOnly="true">
+			<options>
+				<option value="WEP">WEP</option>
+				<option value="WPA">PSK</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="authentication-type">
+		<item-type>String</item-type>
+		<label>Wifi Authentication Type</label>
```

Comment:
Rather `authentication` with label "Authentication". and so on for the other channel types.